### PR TITLE
Functionality: Themify navigation icons

### PR DESCRIPTION
**Pull request by Soitora**
Pretty bold change.

I had to use Vue Themify as I could not figure out how to import components from another file.
Here it's just setting an attribute so that works as planned but Material Icons is an entire component, calling it just prints it literally.

Icon list: https://z3by.github.io/vue-themify-icons/icons/

### CODE REVIEW

File: src/.vuepress/theme/components/NavLinks.vue [hunk 1]
```diff
@@ -0,0 +1,172 @@
+<template>
```
Comment (arkon) [inline src/.vuepress/theme/components/NavLinks.vue:1]: Indentations seems off in this file. Are all of these ejected files or something?

Comment (Soitora) [reply src/.vuepress/theme/components/NavLinks.vue:1]: They are

**Comment by arkon** [comment]
Was I supposed to review this again? I think I substituted "Experimentation" with "WIP" when I looked at it before.

**Comment by Soitora** [comment]
Uhh I really don't remember, it has been a while ^^
Might need to rebase it and look it over again

**Pull request closed without merging**